# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/sway/default-xterm.patch
# Copyright (C) 2023 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- sway-1.8.1/config.in.vanilla	2023-05-17 22:51:14.396707889 +0200
+++ sway-1.8.1/config.in	2023-05-17 22:51:45.039707047 +0200
@@ -14,7 +14,7 @@
 set $up k
 set $right l
 # Your preferred terminal emulator
-set $term foot
+set $term xterm
 # Your preferred application launcher
 # Note: pass the final command to swaymsg so that the resulting window can be opened
 # on the original workspace that the command was run on.
